scripts/undist.ts

```diff
@@ -0,0 +1,80 @@
+import yargs from "yargs";
+import { fileURLToPath } from "node:url";
+import fs from "node:fs";
+import YAML from "yaml";
+
+const argv = yargs(process.argv.slice(2))
+  .scriptName("undist")
+  .usage("$0 path", "Generate .yml from .yml.dist", (yargs) =>
+    yargs.positional("path", {
+      type: "string",
+      describe: "Path to feature.yml.dist file",
+      demandOption: true,
+    }),
+  )
+  .option("add", {
+    alias: "a",
+    type: "array",
+    describe: "BCD keys to add to the feature.yml file",
+  })
+  .option("sort", {
+    alias: "s",
+    type: "boolean",
+    default: true,
+    describe: "Sort the compat_features array",
+  }).argv;
+
+// Support `path/to/feature.yml.dist`, `path/to/feature.yml`, or `path/to/feature`
+function getPaths(path: string): { yml: string; dist: string } {
+  let base = path;
+  if (path.endsWith(".yml.dist")) {
+    base = path.slice(0, -9);
+  } else if (path.endsWith(".yml")) {
+    base = path.slice(0, -4);
+  }
+  return {
+    yml: `${base}.yml`,
+    dist: `${base}.yml.dist`,
+  };
```

Comment:
Importing [`node:path`](https://nodejs.org/api/path.html) could avoid some magic numbers. Something like this:

```suggestion
  const { name, ext } = path.parse(path);
  const id = ext === ".dist" ? path.basename(name) : name;

  return {
    yml: `${id}.yml`,
    dist: `${id}.yml.dist`,
  };
```